packages/core/core-flows/src/cart/steps/compensate-payment-if-needed.ts

```diff
@@ -51,20 +48,7 @@ export const compensatePaymentIfNeededStep = createStep(
       }
     )
 
-    if (paymentSession.status === PaymentSessionStatus.AUTHORIZED) {
-      try {
-        await paymentModule.cancelPayment(paymentSession.id)
```

Comment:
comment: This seems to have been in a broken state up until now, since `cancelPayment` requires a payment ID, not a session ID. If we revert to keep this, will you update this? 

I think we should be able to do `paymentSession.payment.id`